tests/python/pytorch/graphbolt/test_itemset.py

```diff
@@ -56,6 +56,41 @@ def __iter__(self):
         _ = len(item_set)
 
 
+def test_ItemSet_iteration():
```

Comment:
seems less is added compared to existing testcase: `test_ItemSet_length`. If so, let's just add the iterate code into existing one.

Reply:
The reason I added a new one is to have a test case that focuses solely on one function. But It would be better to polish the naming of testcase.

Reply:
But I cannot distinguish such focus as it's almost the same as existing one. looks duplicate to me. the iteration logic was supposed to be added into existing one.

Reply:
Done